modules/transformations/snowflake/sql/ST_BUFFER.sql

```diff
@@ -0,0 +1,42 @@
+----------------------------
+-- Copyright (C) 2022 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION _BUFFER
+(geojson STRING, distance DOUBLE, segments DOUBLE)
+RETURNS STRING
+LANGUAGE JAVASCRIPT
+IMMUTABLE
+AS $$
+    if (!GEOJSON || DISTANCE == null || SEGMENTS == null) {
+        return null;
+    }
+
+    const options = {
+        units: 'meters',
+        steps: Number(SEGMENTS)
+    };
```

Comment:
Is not something meaningful at all. But in the rest of UDFs we are declaring options after @@SF_LIBRARY@@